bluesky_queueserver/manager/comms.py

```diff
@@ -947,6 +1047,12 @@ def zmq_single_request(method, params=None, *, zmq_server_address=None):
     params: dict or None
         Dictionary of parameters (payload of the message). If ``None`` then
         the message is sent with empty payload: ``params = {}``.
+    server_public_key: str or None
+        Server public key (z85-encoded 40 character string). Valid public key from server
```

Comment:
```suggestion
        Server public key (z85-encoded 40 character string). The valid public key from the server's
```

Reply:
I think that the `server` has to be without `'s` here.

Reply:
It's fine either way, but I wanted to stress the belonging of the `public/private key pair` to the `server`. First time I read this sentence I had to re-read it to understand what it meant.

Reply:
I updated the suggestion here to be compatible with the 2 other places above.